browsers/chrome.json

```diff
@@ -793,13 +793,13 @@
           "engine_version": "113"
         },
         "114": {
-          "release_date": "2023-05-30",
+          "release_date": "2023-05-24",
```

Comment:
Please separate out this change and the change to chrome_android.json to a new PR.